frontend/src/components/page_content/profile/family.js

```diff
@@ -62,15 +68,15 @@ class Family extends React.Component {
               defaultValue={this.props.partners[0].firstName}
               onChange={this.setPartner}
             />
-            <ToastInput
-              type="number"
+            <ToastSelect
+              options={birthYearOptions}
+              name="partnerbirthYear"
               label="Partner's Birth Year"
-              name="birthYear"
```

Comment:
we need name='birthYear' or it won't update in the reducer (look at the setPartner function)